save-agent/src/linuxX64Main/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -91,7 +91,7 @@ class SaveAgent(internal val config: AgentConfiguration,
                 sendHeartbeat(ExecutionProgress(0))
             }
             if (response.isSuccess) {
-                when (val heartbeatResponse = response.getOrNull().also {
+                when (val heartbeatResponse = response.getOrThrow().also {
```

Comment:
This fixes compiler warning about non-exhaustive `when`, because there should be a separate branch for `null`. However, we check `response.isSuccess`, so `null` will never be returned.